### DISCUSSION TITLE
Request Entity Too Large

### DISCUSSION THREAD

**Discussion by CosmasSC128** [Budibase/budibase#4912] [Help] [answered]
Budibase docs say uploader can take 20mb, but I'm getting request entity too large on files far smaller than that. I get it for a 1.1mb file example. Any idea what could be going on, or a work around?

**Comment by xnohat** [comment, marked as answer]
I have same problem from Nginx reverse proxy, just add line below to fix

http {
    client_max_body_size 100M;
}

**Reply by CosmasSC128** [reply]
Thanks for the information.
